apps/src/templates/instructions/ReviewTab.jsx

```diff
@@ -265,12 +291,28 @@ const styles = {
     width: '100%',
     display: 'flex',
     justifyContent: 'flex-end',
-    flexDirection: 'column',
-    margin: '10px 0'
+    flexDirection: 'column'
   },
   checkboxErrorMessage: {
     fontStyle: 'italic',
     textAlign: 'end',
     fontSize: '12px'
+  },
+  messageText: {
+    fontSize: '18px',
+    marginBottom: '25px',
+    textAlign: 'center',
+    fontStyle: 'italic',
+    color: '#818181'
```

Comment:
Would one of the colors here work for this gray? https://github.com/code-dot-org/code-dot-org/blob/staging/shared/css/color.scss

Reply:
If yes, it can be used like this: https://github.com/code-dot-org/code-dot-org/blob/39a13de0554776e43dea7552b15bb1d0e85662e5/apps/src/code-studio/components/libraries/LibraryViewCode.jsx#L6

Reply:
This is helpful too for stuff like this:

https://code-dot-org.github.io/cdo-styleguide/?selectedKind=Colors&selectedStory=All%20of%20them&full=0&addons=1&stories=1&panelRight=0&addonPanel=storybook%2Factions%2Factions-panel

Reply:
Ah nice, thanks! These are helpful - I didn't find the exact same grey in this file, but I can use something similar if it's not noticeably different or add this to `colors.scss`